lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -17,19 +17,52 @@ module Cardano.Wallet.Api
 
       -- * Shelley
     , Wallets
+        , DeleteWallet
+        , GetWallet
+        , ListWallets
+        , PostWallet
+        , PutWallet
+        , PutWalletPassphrase
+        , GetUTxOsStatistics
+
     , Addresses
+        , ListAddresses
+
     , CoinSelections
+        , SelectCoins
+
     , Transactions
+        , CreateTransaction
+        , PostTransactionFee
+        , ListTransactions
+        , DeleteTransaction
+
     , StakePools
+        , ListStakePools
+        , JoinStakePool
+        , QuitStakePool
+        , DelegationFee
 
     -- * Byron
     , ByronWallets
+        , DeleteByronWallet
+        , GetByronWallet
+        , ListByronWallets
+        , PostByronWallet
+
     , ByronTransactions
+        , ListByronTransactions
+        , DeleteByronTransaction
+
     , ByronMigrations
+        , MigrateByronWallet
+        , GetByronWalletMigrationInfo
 
     -- * Miscellaneous
     , Network
+
     , Proxy_
+        , PostExternalTransaction
```

Comment:
I can see what you're attempting to do here (with nested indentation).

Though when documentation gets rendered with Haddock, the exported entries will still appear as a linear list, without any nesting.

Perhaps, in that case, it would be nicer to use Haddock to indicate nesting?

Reply:
Could be, I haven't spent much time on the Haddock output _yet_. I'll see what we can do here too :thinking: